app/src/main/java/ch/epfl/favo/user/UserUtil.java

```diff
@@ -27,18 +33,15 @@ public static UserUtil getSingleInstance() {
   }
 
   /**
-   * @param user Corresponds to user in db.
-   * @param pw Corresponds to their password in db.
-   * @throws IllegalArgumentException Should check for invalid inputs.
-   * @throws NotImplementedException Temporary exception while we implement.
+   * @param user A user object.
+   * @throws RuntimeException Unable to post to DB.
    */
-  public void createAccount(String user, String pw)
-      throws IllegalArgumentException, NotImplementedException {
-    /*
-    TODO: Use Firebase API to fetch user data. \
-     Upload it to DB and return to welcome screen for login
-     */
-    throw new NotImplementedException();
+  public void postAccount(User user) throws RuntimeException {
```

Comment:
I think in the future these void functions should return a Task that can be checked for completion and used to notify the UI

Reply:
Agreed, I need more help with tasks/futures so ill work with alex on it this next sprint